FlowCrypt/Functionality/Services/Account Server Services/EnterpriseServerApi.swift

```diff
@@ -79,8 +79,8 @@ class EnterpriseServerApi: EnterpriseServerApiType {
 
             return urlString
         } catch {
-            if let httpError = error as? HttpErr,
-               let nsError = httpError.error as NSError?,
+            if let apiError = error as? ApiError,
```

Comment:
@tomholub 
Maybe it is a bad idea to check tolerated `NSError` codes at business logic level?

Reply:
Maybe. It's not the worst. Please add a convenience method like `self.shouldTolerateWhenCallingOpportunistically(error: Error)`. Because I'd like to later improve that method with #414 . Enterprise build will not be tolerating missing FES (later) and that convenience method will be a good place to put it.